modules/library/src/main/scala/zio/elasticsearch/executor/HttpExecutor.scala

```diff
@@ -44,7 +45,7 @@ import zio.elasticsearch.request.{CreationOutcome, DeletionOutcome, UpdateOutcom
 import zio.elasticsearch.result._
 import zio.json.ast.Json
 import zio.json.ast.Json.{Arr, Obj, Str}
-import zio.json.{DeriveJsonDecoder, JsonDecoder}
+import zio.json.{DeriveJsonDecoder, JsonDecoder, _}
```

Comment:
Does it make sense to use `import zio.json._`?

Reply:
executeBulk implementation is based on sendRequest implementation, it’s zio.json._ to parse body into BulkResponse. 
=> a better solution is to use sendRequestWithCustomerResponse